espnet2/mt/espnet_model.py

```diff
@@ -50,6 +50,7 @@ def __init__(
         report_bleu: bool = True,
         sym_space: str = "<space>",
         sym_blank: str = "<blank>",
+        patch_size: int = 1,
```

Comment:
what is the `patch_size`? Can you add an explanation to the appropriate place?

Reply:
Update the comments accordingly